syntax = "proto3";

message EmailUser {
    string username = 1;
    string email = 2;
    string imageUrl = 3;
    bool is_active = 4;
    bool is_verified = 5;
    string role = 6;
}